packages/node_modules/cerebral/src/views/View.js

```diff
@@ -5,8 +5,11 @@ import {
   throwError,
   ensureStrictPath,
   createResolver,
+  getStateTreeProp,
 } from './../utils'
 
+let verifyPropsWarned = false
```

Comment:
it should be like https://github.com/cerebral/cerebral/pull/953/files#diff-90e0cc5d48b146c1d2708838e101fc6dR39